cabal-install/changelog

```diff
@@ -66,6 +66,8 @@
 	* Paths_ autogen modules now compile when `RebindableSyntax` or
 	`OverloadedStrings` is used in `default-extensions`.
 	[stack#3789](https://github.com/commercialhaskell/stack/issues/3789)
+    * getDataDir` and other `Paths_autogen` functions now work correctly
```

Comment:
I put this in the `2.2.0.0` release notes, as that was the most recent one listed. If this won't make it until the next release, let me know and I can make a section for `2.x.0.0` (where `x` is presumably 4).